lib/ClusterShell/CLI/Display.py

```diff
@@ -137,6 +151,42 @@ def __init__(self, options, config=None, color=None):
             if hasattr(options, 'debug') and options.debug:
                 self.verbosity = VERB_DEBUG
 
+    def _has_cli_color(self):
+        """Tests CLICOLOR environment variable to determine wether to
+        use color or not on output.
+        """
+        # When CLICOLOR_FORCE is set to something else than 0
+        # colors must be used.
+        if os.getenv("CLICOLOR_FORCE", "0") != "0":
+            return True
+
+        cli_color = os.getenv("CLICOLOR")
+
+        if cli_color is None:
```

Comment:
Looks like:

    if cli_color is None:
        return None
    elif cli_color != "0":
        return sys.stdout.isatty()
    else:
        return False